### PR TITLE
Migrating incident process to current docs

### PR DESCRIPTION
**Pull request by julialawrence**
# Pull Request Objective

This piece of work is being tracked in
[this](https://github.com/ministryofjustice/analytical-platform-ithc-2024/issues/25)
GitHub Issue.

Migrating/updating our incident process documentation from silver-dollop to current docs.

## Checklist

> [!NOTE]
> Each items should be checked. Skipping below checks could delay your PR review!

- [X] I have reviewed the [style guide](https://docs.analytical-platform.service.justice.gov.uk/documentation/platform/infrastructure/terraform.html#terraform)
and ensured that my code complies with it
- [X] All checks have passed (or override label applied, if I've
used the `override-static-analysis` label, I've explained why)
- [X] I have self-reviewed my code
- [X] I have reviewed the checks and can attest they're as expected

### CODE REVIEW

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+title: Incident Process
+last_reviewed_on: 2024-11-12
+review_in: 3 months
+---
+
+# Incident Process
+
+This document describes our incident handling process for network, operations and security issues.
+
+> This document assumes the user has a PagerDuty account and is part of the Ansalytical Platform team.  To use the in Slack features the user must authorise the PagerDuty app in Slack.
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:12]: ✏️ typo

``` suggestion
> This document assumes the user has a PagerDuty account and is part of the Analytical Platform team.  To use the in Slack features the user must authorise the PagerDuty application in Slack.
```

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+> If this incident constitutes a security breach, you must also report it [here](https://goo.gl/forms/frsB1h8AGv3Zefwq2)
+
+We define an incident as an event which:
+
+* is unplanned, and
+* impacts end users or our direct users (developers and engineers), or
+* degrades user-facing services, or
+* increases risk to production services
+
+If this event does not constitute an incident, the appropriate response is ask a user to [raise an support issue](https://github.com/ministryofjustice/data-platform-support/issues/new/choose) or [raise a ticket](https://github.com/ministryofjustice/analytical-platform/issues/new/choose) in our GitHub repository, however if the issue is security related, it is [raised here](https://github.com/ministryofjustice/analytical-platform-ithc-2024/issues/new?template=Blank+issue)
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:25]:
```suggestion
If this event does not constitute an incident, the appropriate response is ask a user to [raise an support issue](https://github.com/ministryofjustice/data-platform-support/issues/new/choose) or [raise a ticket](https://github.com/ministryofjustice/analytical-platform/issues/new/choose) in our GitHub repository, however if the issue is security related, it is [raised here](https://github.com/ministryofjustice/analytical-platform-ithc-2024/issues/new?template=Blank+issue).
```

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+#### Which Impact to assign?
+
+| **Impact** | Description |
+|---|---|
+|Major|The whole platform is down or unavailable, all user applications are unavailable|
+|Minor|Part of the platform is down or unavailable, some user applications are impacted or unavailable|
+|All Good|Incident has been resolved, all service is restored|
+
+Select Time till Next Update. Note that users should be kept regularly updated especially during major outages, and choose accordingly.
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:53]:
``` suggestion
Select `Time till Next Update`. Note that users should be kept regularly updated especially during major outages, and choose accordingly.
```

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+Tick the **Send update notifications to subscribers** box.
+
+Click **Post** button once you are happy with the form. This will update the external page. The PagerDuty Status Page slack integration will then automatically post the status in the [analytical-platform-support](https://moj.enterprise.slack.com/archives/C4PF7QAJZ) and [ask-data-engineering](https://moj.enterprise.slack.com/archives/C8X3PP1TN) channels.
+
+## 3. Managing an Incident
+
+Incident communications must be maintained in a single slack thread in the main [analytical-platform](https://moj.enterprise.slack.com/archives/C04M8224WCV) team channel for maximum visibility.
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:61]:
```suggestion
Incident communications must be maintained in a single Slack thread in the main [analytical-platform](https://moj.enterprise.slack.com/archives/C04M8224WCV) team channel for maximum visibility.
```

**Review by jacobwoffenden — APPROVED** [review]

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+### 3.1 Starting a Thread
+
+Incident slack thread should have a clear starting message and a thread emoji to ensure consistency. Ex: "Airflow Autoscaling Incident :thread:"
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:65]:
``` suggestion
Incident Slack thread should have a clear starting message and a thread emoji to ensure consistency. Ex: "Airflow Autoscaling Incident :thread:"
```

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+Please note, communications in the Slack thread could later be used to reconstruct the timeline for post-incident report (if required) therefore it is critical to keep all comms to the single incident thread.
+
+> Regular updates to the external status page should be posted to allow stakeholder visibility.
+
+## 4. Fix the problem
+
+Please bear in mind that not every incident requires the whole team to be involved (even if they all *want* to join in).
+
+Log a support ticket if necessary
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:77]:
``` suggestion
Log a support ticket if necessary.
```

File: source/documentation/runbooks/004-managing-an-incident.html.md.erb [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+If the incident cannot be resolved within the team or if the issue lies with a 3rd party log a support ticket with the 3rd party.
```
Comment (Gary-H9) [inline source/documentation/runbooks/004-managing-an-incident.html.md.erb:79]:
```suggestion
If the incident cannot be resolved within the team or if the issue lies with a third-party log a support ticket with the third-party.
```

**Review by Gary-H9 — COMMENTED** [review]
Too late 😢

**Pull request merged**